pkg/ccl/backupccl/testdata/backup-restore/schedule-privileges

```diff
@@ -122,21 +122,10 @@ exec-sql user=testuser
 DROP SCHEDULE $fullID
 ----
 
-query-sql
-with schedules as (SHOW SCHEDULES FOR BACKUP) SELECT label,command,owner FROM schedules WHERE id IN
-($fullID, $incID) ORDER BY next_run;
-----
-foocluster_admin BACKUP INTO LATEST IN 'external://foo/cluster' WITH detached root
```

Comment:
this query should work, but it will return nothing, correct?

Reply:
When I was testing it I believe it was throwing errors, I can double check